platforms/software/antlr/src/main/java/org/gradle/api/plugins/antlr/AntlrPlugin.java

```diff
@@ -90,6 +90,7 @@ public void execute(final SourceSet sourceSet) {
                     // 4) Register a source-generating task, and
                     TaskProvider<AntlrTask> antlrTask = project.getTasks().register(taskName, AntlrTask.class, task -> {
                         task.setDescription("Processes the " + sourceSet.getName() + " Antlr grammars.");
+                        task.setGroup("Antlr");
```

Comment:
Most of our other groups are lowercase. We should make this lowercase as well.

https://github.com/gradle/gradle/blob/b6dfa822a751b5bfd4778015b83fd6d2ce5d287a/platforms/software/platform-base/src/main/java/org/gradle/language/base/plugins/LifecycleBasePlugin.java#L34-L39

Reply:
Applied the feedback : )